python/doc/examples/reliability_sensitivity/sensitivity_analysis/plot_sensitivity_wingweight.py

```diff
@@ -0,0 +1,462 @@
+# For each 2D cross cut, the other variables are fixed to the input distribution mean values.
+# This graph allows to have a first idea of the variations of the function in pair of dimensions.
+# The colors of each contour plot are comparable. The number of contour levels are related to the amount of variation of the function in the corresponding coordinates.
+
+fig = plt.figure(figsize=(12, 12))
+lowerBound = m.distributionX.getRange().getLowerBound()
+upperBound = m.distributionX.getRange().getUpperBound()
+
+# Definition of number of meshes in x and y axes for the 2D cross cut plots
+nX = 20
+nY = 20
+for i in range(m.dim):
+    for j in range(i):
+        crossCutIndices = []
+        crossCutReferencePoint = []
+        for k in range(m.dim):
+            if k != i and k != j:
+                crossCutIndices.append(k)
+                # Definition of the reference point
+                crossCutReferencePoint.append(m.distributionX.getMean()[k])
+        # Definition of 2D cross cut function
+        crossCutFunction = ot.ParametricFunction(
+            m.model, crossCutIndices, crossCutReferencePoint)
+        crossCutLowerBound = [lowerBound[j], lowerBound[i]]
+        crossCutUpperBound = [upperBound[j], upperBound[i]]
+        # Definition of the mesh
+        inputData = ot.Box([nX, nY]).generate()
+        inputData *= (ot.Point(crossCutUpperBound)-ot.Point(crossCutLowerBound))
+        inputData += ot.Point(crossCutLowerBound)
+        meshX = np.array(inputData)[:,0].reshape(nX+2,nY+2)
+        meshY = np.array(inputData)[:,1].reshape(nX+2,nY+2)
+        data = crossCutFunction(inputData)
+        meshZ = np.array(data).reshape(nX+2,nY+2)
+        levels = [(150 + 3*i) for i in range(101)]
+        graph = ot.Graph()
+        
+        # Creation of the contour        
+        index = 1 + i * m.dim + j
+
+        ax = fig.add_subplot(m.dim, m.dim, index)
+        ax.contour(meshX, meshY,meshZ,levels,cmap='hsv')
+        ax.set_xticks([])
+        ax.set_yticks([])        
+        
+        # Creation of axes title
+        if j==0:
+            ax.set_ylabel(m.distributionX.getDescription()[i])            
+        if i ==9:
+            ax.set_xlabel(m.distributionX.getDescription()[j])
+
+# %%       
+# We can see that the variables :math:`t_c, N_z, A, W_{dg}` seem to be influent on the wing weight whereas :math:`l, W_p, W_{fw}` do not have influence on the function.
+            
+
+# %%
+# Data generation 
+# ---------------
+#
+# We create the input and output data for the estimation of the different sensitivity coefficients and we get the input variables description:
+
+inputNames = m.distributionX.getDescription()
+
+size = 500
+inputDesign = m.distributionX.getSample(size)
+outputDesign = m.model(inputDesign)
+
+# %%
+# Let's estimate the PCC, PRCC, SRC, SRRC, Pearson and Spearman coefficients, display and analyze them.
+# We create a `CorrelationAnalysis` model.

```

Comment:
```suggestion
# We create a :class:`~openturns.CorrelationAnalysis` model.

```
To get a link to the relevant API page.